src/olympia/abuse/models.py

```diff
@@ -133,6 +134,7 @@ class CinderJob(ModelBase):
     )
     decision_id = models.CharField(max_length=36, default=None, null=True, unique=True)
     decision_date = models.DateTimeField(default=None, null=True)
+    decision_notes = models.CharField(max_length=255, default=None, null=True)
```

Comment:
Per slack thread make that a `TextField` with a `max_length=1000`, and truncate incoming data at `max_length`.